aiida_common_workflows/workflows/relax/abinit/generator.py

```diff
@@ -23,8 +23,22 @@ class AbinitRelaxInputsGenerator(RelaxInputsGenerator):
     _default_protocol = 'moderate'
     _calc_types = {'relax': {'code_plugin': 'abinit', 'description': 'The code to perform the relaxation.'}}
     _relax_types = {
-        RelaxType.ATOMS: 'Relax only the atomic positions while keeping the cell fixed.',
-        RelaxType.ATOMS_CELL: 'Relax both atomic positions and the cell.'
+        RelaxType.NONE:
+        'Fix the atomic positions and volume and shape of the cell.',
+        RelaxType.ATOMS:
+        'Relax the atomic positions while keeping the volume and shape of the cell fixed.',
+        RelaxType.VOLUME:
+        'Relax the volume of the cell while keeping its shape and the atomic positions fixed.',
```

Comment:
Relax the volume of the cell while keeping its shape and relative atomic positions fixed.